CMSgov/design-system
# [NO-TICKET] USWDS design tokens exploration

## Summary
USWDS design tokens exploration.

### Added
- USWDS spacing design tokens

## How to test
http://design-system-demo.s3-website-us-east-1.amazonaws.com/feature/uswds-design-tokens/

- Test CMSDS works as expected
- Test with Child design system by updating the package.json to point to this branch:
  - [Hcgov DS test branch: WNMGDS-642/use-uswds-spacing-tokens](https://github.com/CMSgov/hcgov-design-system/tree/WNMGDS-642/use-uswds-spacing-tokens)
  - [Mgov DS test branch: WNMGDS-642/use-uswds-spacing-tokens](https://github.com/CMSgov/mgov-design-system/tree/WNMGDS-642/use-uswds-spacing-tokens)

```
package.json file:
"@cmsgov/design-system": "file:../design-system/packages/design-system",
"@cmsgov/design-system-docs": "file:../design-system/packages/design-system-docs",
"@cmsgov/design-system-scripts": "file:../design-system/packages/design-system-scripts",
```
